gazebo_ros/scripts/spawn_model

```diff
@@ -288,6 +288,7 @@ if __name__ == "__main__":
         print usage()
     else:
         print("spawn_model script started") # make this a print incase roscore has not been started
+        rospy.init_node('spawn_model')
```

Comment:
To clarify... the `init_node` call originally exists on line 197, does it make a difference whether `rospy.init_node(...)` is called before or after line 199-200 block?  What happens if we left `rospy.init_node(...)` where it was before?